app/models/planning_application.rb

```diff
@@ -542,7 +542,8 @@ def updates_required?
     assessment_details_for_review.any?(&:update_required?) ||
       permitted_development_right&.update_required? ||
       policy_classes.any?(&:update_required?) ||
-      (committee_decision.present? && committee_decision.current_review.rejected?)
+      (committee_decision.present? && committee_decision.current_review.rejected?) ||
+      neighbour_review_requested?
```

Comment:
Something to look at in the next state of work is whether there's a better way to do this such as a `updates_required` boolean column that gets set to true in a callback.